### ISSUE TITLE
Any chance of Linux support?

### ISSUE THREAD

**Issue by EmeraldSnorlax** [Kataiser/tf2-rich-presence#43]
Seeing as its all python it should be relatively easy to port -- trying to run build.py or main.py just crashes on me -- perhaps because of using \ for paths instead of / ?
Output of tests.py is [here](https://pastebin.com/jYRgWZrd)

**Comment by Kataiser** [comment]
Yeah, tests and build use backslashes all over the place, but the bulk of the main code should be close to compatible with Linux/MacOS (using os.path functions). As I don't have either OS, I haven't tested them for real. Also note that most of the code has been made specifically for the built releases, which means that I have little to no idea how well it works as it's organized in the repo. 

While you're at it, may I see the crash for main.py?

**Comment by EmeraldSnorlax** [comment]
Sure! Here: https://pastebin.com/c394VmL0

**Comment by Kataiser** [comment]
Ah yes, forward slashes and backslashes in the same path lol

**Comment by EmeraldSnorlax** [comment]
Going down the line and fixing the errors it complains about eventually leaves me off here after running main.py: https://pastebin.com/Xmvwdez7

(ps by fix i mean using a hacky solution and just replacing the `\` with a `/` :^) )

It starts, but doesn't want to do much else

**Comment by Kataiser** [comment]
`tasklist` is a Windows only command apparently. Honestly, getting the file stuff working is probably the tip of the iceberg considering how much process scanning is going on in this project. At least psutil is cross platform...

**Comment by Kataiser** [comment]
yo thanks :)

**Comment by EmeraldSnorlax** [comment]
if its too much of a hassle, dont worry about it, i can live without telling the whole world that im a friendly murderer on 2Fort

**Comment by Jan200101** [comment]
![image](https://user-images.githubusercontent.com/15076013/66589388-b1a86f80-eb8e-11e9-8eb0-5235266d3ce2.png)
Manged to get it working while working on #53 
#53 could be completed once I know the names of the processes

**Comment by Kataiser** [comment]
Nice!

**Comment by EmeraldSnorlax** [comment]
~~actually, dont worry about it, you can close this if you like. i ended up making my own script for it instead.~~
